gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/ConnectionPool.java

```diff
@@ -456,8 +491,8 @@ private Connection waitForConnection(final long timeout, final TimeUnit unit) th
         // message written to the log should provide some insight into the state of the connection pool telling the
         // user if the pool was running at maximum.
         final String timeoutErrorMessage = String.format(
-                "Timed-out (%s %s) waiting for connection on %s%s%s",
-                timeout, unit, host, System.lineSeparator(), this.getPoolInfo());
+                "Timed-out (%s %s) waiting for connection on %s%s%s. ",
+                timeout, unit, host, System.lineSeparator(), this.getPoolInfo()) + getAdditionalTimeoutInformation();
```

Comment:
Should we break `getAdditionalTimeoutInformation()` into a new line? Since it looks like `this.getPoolInfo()` will output a bunch of information, it may be easier to read?